server/backend/database/memory/database.go

```diff
@@ -559,6 +584,29 @@ func (d *DB) FindDeactivateCandidates(
 	return infos, nil
 }
 
+// FindDeactivateCandidates finds the clients that need housekeeping.
+func (d *DB) FindDeactivateCandidates(
+	ctx context.Context,
+	candidatesLimit int,
+) ([]*database.ClientInfo, error) {
+	projects, err := d.listAllProjectInfos(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	var candidates []*database.ClientInfo
+	for _, project := range projects {
+		infos, err := d.findDeactivateCandidatesPerProject(ctx, project, candidatesLimit)
```

Comment:
The meaning of `candidatesLimit` has changed to limit per project. How about changing the variable name to `candidatesLimitPerProject`?

Reply:
Thank you for your suggestion.
`candidatesLimit` has changed to `candidatesLimitPerProject` to clarify the meaning.